content/docs/tutorials/ocm-and-gitops/_index.md

```diff
@@ -7,4 +7,8 @@ draft: false
 images: []
 sidebar:
   collapsed: true
----
\ No newline at end of file
+---
+
+Just GitOps. With OCM as underlying model, the OCM Controllers as the execution engine, and their seamless integration into FluxCD you can manage your GitOps workflows with OCM.  
```

Comment:
This sentence looks off. Why is it "Just GitOps"?

Reply:
I commited `Seamless GitOps integration. OCM provides the underlying model and controllers that integrate naturally with FluxCD, empowering you to manage sophisticated GitOps workflows with confidence.` now.